### PR TITLE
Adding flexpolyline

### PR DESCRIPTION
**Pull request by sackh**
Signed-off-by: Kharude, Sachin <sachin.kharude@here.com>

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sackh** [comment]
@conda-forge-linter

**Comment by sackh** [comment]
@conda-forge/core  - will you please help. I am not sure why lint is failing.

**Comment by ocefpaf** [comment]
The logs mention a `meta.yaml/` directory, that is odd :-/

File: recipes/flexpolyline/meta.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ test:
 
 about:
   home: https://here.com
-  summary: Flexible Polyline encoding a lossy compressed representation of a list of coordinate pairs or triples.
+  summary: Flexible Polyline encoding a lossy compressed representation of a list of coordinate pairs or triples
   dev_url: https://github.com/heremaps/flexible-polyline
   license: MIT
   license_file: LICENSE
```
[the remark this answers is not in the document]
Comment (sackh) [reply recipes/flexpolyline/meta.yml:38]: Thanks @viniciusdc for your reply, I added the LICENSE file, but it still fails with error - `NotADirectoryError: [Errno 20] Not a directory: '/home/conda/staged-recipes-copy/recipes/flexpolyline/meta.yml/meta.yaml'`

**Comment by wolfv** [comment]
can you rename the file to `meta.yaml` (not `meta.yml`)?

**Comment by sackh** [comment]
> can you rename the file to `meta.yaml` (not `meta.yml`)?

sorry, my bad. now the lint issue is fixed. Thank you.

**Comment by sackh** [comment]
@conda-forge-admin, please ping team

**Pull request merged**